.github/workflows/shellcheck-lint.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: ShellCheck Lint
+on: [push, pull_request]  # yamllint disable-line rule:truthy
+
+jobs:
+  build:
+    name: Lint ShellCheck
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Lint ShellCheck
+        uses: ludeeus/action-shellcheck@master
+        env:
+          SHELLCHECK_OPTS: -e SC2181 -e SC3010 -e SC3014
```

Comment:
I'd prefer to have those not excluded. especially SC3010 and SC3014 are pretty important warnings. Instead of deactivate them globally, I'd prefer to deactivate them per shellcheck directive in the scripts, if even necessary.

Reply:
Have a look at the first lines of https://github.com/freifunk-berlin/falter-builter/pull/114/files to see how to use those directives.

Reply:
I moved the exceptions for the directives to the files, added Todos in the script that already has Todos and fixed the string comparison in roles/cfg_openwrt/files/common/init.d/replace_secrets. Please have a look.